odbc/src/tests.rs

```diff
@@ -1,10 +1,16 @@
 use crate::{
-    handles::{Connection, ConnectionState, Env, EnvState, MongoHandle, Statement, StatementState},
-    SQLAllocHandle, SQLFreeHandle,
+    handles::{
+        Connection, ConnectionState, Descriptor, Env, EnvState, MongoHandle, ODBCError, Statement,
+        StatementState,
+    },
+    SQLAllocHandle, SQLFreeHandle, SQLGetDiagRecW,
 };
 use odbc_sys::{Handle, HandleType, SqlReturn};
 use std::sync::RwLock;
 
+const ERROR_MESSAGE_NULL: &str = "[MongoDB][API]SQLDrivers\0";
```

Comment:
What does the suffix NULL mean for this 2 const variables?

Reply:
"_NULL" means the string is null terminated. I previously had other constants for non-null terminated strings, which is why I introduced this suffix, but I can delete it now that I've removed those constants.